client/components/select-dropdown/test/index.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+import { assert } from 'chai';
+import { shallow, mount } from 'enzyme';
```

Comment:
Ok, I won't abuse of `mount` in the future.
I'll check if I can get equivalent assertions without mounting the whole component... otherwise I'll keep this one as is.